lib/libutee/include/tee_api.h

```diff
@@ -114,7 +114,7 @@ bool TEE_MaskCancellation(void);
 TEE_Result TEE_CheckMemoryAccessRights(uint32_t accessFlags, void *buffer,
 				       uint32_t size);
 
-void TEE_SetInstanceData(void *instanceData);
+void TEE_SetInstanceData(const void *instanceData);
```

Comment:
I've looked at the GP API and the APIs you did not change. Some API could also be "fixed": 
```
TEE_Realloc(const void *buffer, uint32_t newSize);`
            ^^^^^

const void *TEE_GetInstanceData(void);`
^^^^^
TEE_Result TEE_PopulateTransientObject(TEE_ObjectHandle object, const TEE_Attribute *attrs, uint32_t attrCount);`
                                                                ^^^^^

 TEE_Result TEE_OpenPersistentObject(uint32_t storageID, const void *objectID, uint32_t objectIDLen, uint32_t flags, TEE_ObjectHandle *object);
                                                         ^^^^^
```